config/billing_rates.yml

```diff
@@ -37,3 +37,4 @@
 - { id: 0fc7f264-87f0-446d-bee7-c0454149b1bc, resource_type: GitHubRunnerMinutes,     resource_family: standard-8-arm,  location: global,         unit_price: 0.0032 }
 - { id: 995ab760-5d97-4a7b-981e-c9947233c78e, resource_type: GitHubRunnerMinutes,     resource_family: standard-16-arm, location: global,         unit_price: 0.0064 }
 - { id: c31b3640-92fa-49c3-bae8-c1011cc5f917, resource_type: GitHubRunnerConcurrency, resource_family: standard,        location: global,         unit_price: 0.0000248016 }
+- { id: 27d84309-e39d-8d78-b603-7632dafe374e, resource_type: GitHubGpuRunnerMinutes,  resource_family: standard-gpu-6,  location: global,         unit_price: 0.0320 }
```

Comment:
Since, you already changed resource family to `standard-gpu-6` from `standard-6`. Why did we change resource_type too? It caused unnecessary code changes

Reply:
This one is the strongest one in my comments.

Reply:
Thanks for implementing it